main.pm

```diff
@@ -626,6 +626,21 @@ sub load_autoyast_tests(){
     #    next boot in load_reboot_tests
 }
 
+sub load_skenkins_tests {
+    my $anyloaded = 0;
+    if (get_var("SLENKINS_NODEFILE")) {
+        my $node = get_var("SLENKINS_NODE");
+        if ($node && $node ne 'control' ) {
+            loadtest "slenkins/slenkins_node.pm";
+        }
+        else {
+            loadtest "slenkins/slenkins_control.pm";
+        }
+        $anyloaded = 1;
+    }
```

Comment:
aehm, what is the purpose of this variable. Just a return 1 here and no return otherwise would be clearer IMO